key-chord.el

```diff
@@ -54,6 +54,12 @@ sure when executing whether two keys were typed quickly or slowly
 when recorded.)"
   :type 'boolean)
 
+(defcustom key-chord-min-delay 0.05
```

Comment:
This new variable isn't being used. I assume you start using it in a later commit. These commits should be squashed into a single commit.

Reply:
Hm you are somehow reviewing an old commit. Yup, I have not cleaned up the commit history. Maybe I should squash them all into 1?